File: android-jsBridge/src/main/java/me/yamlee/jsbridge/widget/dialog/internel/EnhanceBtn.java
package me.yamlee.jsbridge.widget.dialog.internel;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class EnhanceBtn {
    public static final int ENHANCE_CONFIRM = 0;//default
    public static final int ENHANCE_CANCEL = 1;
    public static final int ENHANCE_NONE = 2;

    @Retention(RetentionPolicy.SOURCE)
    @IntDef({ENHANCE_CONFIRM, ENHANCE_CANCEL, ENHANCE_NONE})
    public @interface EnhanceBtnDef {

    }
}

File: android-jsBridgeAll/src/main/java/me/yamlee/bridge/ui/dialog/internel/EnhanceBtn.kt
package me.yamlee.bridge.ui.dialog.internel

import android.support.annotation.IntDef

object EnhanceBtn {
    const val ENHANCE_CONFIRM = 0//default
    const val ENHANCE_CANCEL = 1
    const val ENHANCE_NONE = 2

    @kotlin.annotation.Retention
    @IntDef(ENHANCE_CONFIRM, ENHANCE_CANCEL, ENHANCE_NONE)
    annotation class EnhanceBtnDef
}